server/forge/bitbucket/bitbucket.go

```diff
@@ -302,8 +302,21 @@ func (c *config) BranchHead(_ context.Context, _ *model.User, _ *model.Repo, _ s
 	return "", forge_types.ErrNotImplemented
 }
 
-func (c *config) PullRequests(_ context.Context, _ *model.User, _ *model.Repo, _ *model.ListOptions) ([]*model.PullRequest, error) {
-	return nil, forge_types.ErrNotImplemented
+// PullRequests returns the pull requests of the named repository.
+func (c *config) PullRequests(ctx context.Context, u *model.User, r *model.Repo, p *model.ListOptions) ([]*model.PullRequest, error) {
+	opts := internal.ListOpts{Page: p.Page, PageLen: p.Page}
+	pullRequests, err := c.newClient(ctx, u).ListPullRequests(r.Owner, r.Name, &opts)
+	if err != nil {
+		return nil, err
+	}
+	result := make([]*model.PullRequest, len(pullRequests))
+	for i := range pullRequests {
```

Comment:
I would prefer to use this style:

```suggestion
	for _, pullRequest := range pullRequests {
```

Reply:
OK nice. Is this something generally preferred in woodpecker-ci project?
Should I update this even it's now approved?

Reply:
I prefer (and I think most other maintainers) prefer that way. Yes, please update. Approving can be done as open threads are blocking and need to be resolved first before proceeding.

Reply:
Done. Updated the iteration type.